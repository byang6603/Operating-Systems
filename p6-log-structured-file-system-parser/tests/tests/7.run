../bin/parse_lfs cat /dir/./c/d/.././../../dir/b/../b/././foo.txt ../sample_files/a.img
