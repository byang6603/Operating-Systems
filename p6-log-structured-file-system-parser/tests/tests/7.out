The Project Gutenberg eBook of Hard Times
    
This ebook is for the use of anyone anywhere in the United States and
most other parts of the world at no cost and with almost no restrictions
whatsoever. You may copy it, give it away or re-use it under the terms
of the Project Gutenberg License included with this ebook or online
at www.gutenberg.org. If you are not located in the United States,
you will have to check the laws of the country where you are located
before using this eBook.

Title: Hard Times

Author: Charles Dickens

Release date: January 1, 1997 [eBook #786]
                Most recently updated: December 18, 2023

Language: English

Credits: Transcribed from the 1905 Chapman and Hall edition by David Price.


*** START OF THE PROJECT GUTENBERG EBOOK HARD TIMES ***




                                HARD TIMES
                                   AND
                           REPRINTED PIECES {0}


                                * * * * *

                            By CHARLES DICKENS

                                * * * * *

             _With illustrations by Marcus Stone_, _Maurice_
                     _Greiffenhagen_, _and F. Walker_

                                * * * * *

                       LONDON: CHAPMAN & HALL, LD.
                    NEW YORK: CHARLES SCRIBNER’S SONS

                                   1905




CONTENTS

      _BOOK THE FIRST_.  _SOWING_
                                    PAGE
               CHAPTER I
_The One Thing Needful_                3
               CHAPTER II
_Murdering the Innocents_              4
              CHAPTER III
_A Loophole_                           8
               CHAPTER IV
_Mr. Bounderby_                       12
               CHAPTER V
_The Keynote_                         18
               CHAPTER VI
_Sleary’s Horsemanship_               23
              CHAPTER VII
_Mrs. Sparsit_                        33
              CHAPTER VIII
_Never Wonder_                        38
               CHAPTER IX
_Sissy’s Progress_                    43
               CHAPTER X
_Stephen Blackpool_                   49
               CHAPTER XI
_No Way Out_                          53
              CHAPTER XII
_The Old Woman_                       59
              CHAPTER XIII
_Rachael_                             63
              CHAPTER XIV
_The Great Manufacturer_              69
               CHAPTER XV
_Father and Daughter_                 73
              CHAPTER XVI
_Husband and Wife_                    79
     _BOOK THE SECOND_.  _REAPING_
               CHAPTER I
_Effects in the Bank_                 84
               CHAPTER II
_Mr. James Harthouse_                 94
              CHAPTER III
_The Whelp_                          101
               CHAPTER IV
_Men and Brothers_                   111
               CHAPTER V
_Men and Masters_                    105
               CHAPTER VI
_Fading Away_                        116
              CHAPTER VII
_Gunpowder_                          126
              CHAPTER VIII
_Explosion_                          136
               CHAPTER IX
_Hearing the Last of it_             146
               CHAPTER X
_Mrs. Sparsit’s Staircase_           152
               CHAPTER XI
_Lower and Lower_                    156
              CHAPTER XII
_Down_                               163
     _BOOK THE THIRD_.  _GARNERING_
               CHAPTER I
_Another Thing Needful_              167
               CHAPTER II
_Very Ridiculous_                    172
              CHAPTER III
_Very Decided_                       179
               CHAPTER IV
_Lost_                               186
               CHAPTER V
_Found_                              193
               CHAPTER VI
_The Starlight_                      200
              CHAPTER VII
_Whelp-Hunting_                      208
              CHAPTER VIII
_Philosophical_                      216
               CHAPTER I